VTEX - Custom Fields API.json

```diff
@@ -0,0 +1,1627 @@
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/orderFormId"
+                    },
+                    {
+                        "$ref": "#/components/parameters/linkedEntity.type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/linkedEntity.id"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "description": "Custom field data to create or update in the orderForm.",
+                                "required": ["name", "value"],
+                                "properties": {
+                                    "name": {
+                                        "type": "string",
+                                        "description": "Custom field name.",
+                                        "example": "Cost Center"
+                                    },
+                                    "value": {
+                                        "type": "string",
+                                        "description": "Custom field value, limited to 50 characters.",
```

Comment:
This field is documented above with a char limit of 22. We must make the information consistent. I am going to try and get the answer from the eng team.